index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Medical core</title>
+  </head>
+  <body class="body">
+    <div class="navbar">
+      <header class="header">
+        <h1 class="header__logo">Medical Core</h1>
+        <ul class="header__navbar">
+          <li class="header__item">Home</li>
+          <li class="header__item">Product</li>
+          <li class="header__item">Pricing</li>
+          <li class="header__item">Contact</li>
+        </ul>
+
+        <ul class="header__leftbtn">
+          <li class="btn">Login</li>
+          <li class="btn btn--active">Join Us</li>
+        </ul>
+      </header>
+
+      <div class="banner">
+        <h4 class="banner__header">Welcome</h4>
+        <p class="banner__description">A Great Place to Receive Care</p>
+        <p class="banner__moto">Get your best looking smile now!</p>
+
+        <ul class="banner__btn">
+          <li class="btnn btn--activee">Join Us</li>
+          <li class="btnn btn--disactive">Learn More</li>
```

Comment:
you are using modifier `btn--activee` without its block. or change `btnn` to `btn`